### PR TITLE
avs 0.11.1. support

### PR DESCRIPTION
**Pull request by rahul-aerospike**
- Incorporated proto files change
- Updated tests

### CODE REVIEW

File: tests/standard/sync/test_admin_client_index_get.py [hunk 1]
```diff
@@ -33,8 +33,8 @@ def test_index_get(session_admin_client, empty_test_case, random_name):
     assert result["hnsw_params"]["m"] == 16
     assert result["hnsw_params"]["ef_construction"] == 100
     assert result["hnsw_params"]["ef"] == 100
-    assert result["hnsw_params"]["batching_params"]["max_records"] == 100000
-    assert result["hnsw_params"]["batching_params"]["interval"] == 30000
+    assert result["hnsw_params"]["batching_params"]["max_index_records"] == 100000
+    assert result["hnsw_params"]["batching_params"]["index_interval"] == 30000
```
Comment (dwelch-spike) [inline tests/standard/sync/test_admin_client_index_get.py:37]: `max_reindex_records` and `reindex_interval` should be added to a test case here.

File: tests/standard/aio/test_admin_client_index_list.py [hunk 2]
```diff
@@ -31,8 +31,8 @@ async def test_index_list(session_admin_client, empty_test_case, random_name):
         assert isinstance(index["hnsw_params"]["m"], int)
         assert isinstance(index["hnsw_params"]["ef_construction"], int)
         assert isinstance(index["hnsw_params"]["ef"], int)
-        assert isinstance(index["hnsw_params"]["batching_params"]["max_records"], int)
-        assert isinstance(index["hnsw_params"]["batching_params"]["interval"], int)
+        assert isinstance(index["hnsw_params"]["batching_params"]["max_index_records"], int)
+        assert isinstance(index["hnsw_params"]["batching_params"]["index_interval"], int)
```
Comment (dwelch-spike) [inline tests/standard/aio/test_admin_client_index_list.py:35]: `max_reindex_records` and `reindex_interval` should be added to a test case here.

File: tests/standard/aio/test_admin_client_index_get.py [hunk 3]
```diff
@@ -27,8 +27,8 @@ async def test_index_get(session_admin_client, empty_test_case, random_name):
     assert result["hnsw_params"]["m"] == 16
     assert result["hnsw_params"]["ef_construction"] == 100
     assert result["hnsw_params"]["ef"] == 100
-    assert result["hnsw_params"]["batching_params"]["max_records"] == 100000
-    assert result["hnsw_params"]["batching_params"]["interval"] == 30000
+    assert result["hnsw_params"]["batching_params"]["max_index_records"] == 100000
+    assert result["hnsw_params"]["batching_params"]["index_interval"] == 30000
```
Comment (dwelch-spike) [inline tests/standard/aio/test_admin_client_index_get.py:31]: `max_reindex_records` and `reindex_interval` should be added to a test case here.

**Review by dwelch-spike — COMMENTED** [review]
A few comments

File: src/aerospike_vector_search/types.py [hunk 4]
```diff
@@ -562,30 +584,34 @@ def _to_pb2(self):
...
-        caching_repr = repr(self.caching_params)
+        caching_repr = repr(self.index_caching_params)
         healer_repr = repr(self.healer_params)
         merge_repr = repr(self.merge_params)
         return (
             f"HnswParams(m={self.m}, ef_construction={self.ef_construction}, "
             f"ef={self.ef}, batching_params={batching_repr}, max_mem_queue_size={self.max_mem_queue_size}, "
-            f"caching_params={caching_repr}, healer_repr={healer_repr}, merge_repr={merge_repr}, enableVectorIntegrityCheck={self.enable_vector_integrity_check} )"
+            f"index_caching_params={caching_repr}, healer_repr={healer_repr}, merge_repr={merge_repr}, enableVectorIntegrityCheck={self.enable_vector_integrity_check}, "
+            f"record_caching_params={self.record_caching_params}  )"
```
[the remark this answers is not in the document]
Comment (rahul-aerospike) [reply src/aerospike_vector_search/types.py]: updated,

File: src/aerospike_vector_search/types.py [hunk 5]
```diff
@@ -562,30 +584,34 @@ def _to_pb2(self):
...
         )
 
     def __str__(self) -> str:
         batching_str = str(self.batching_params)
-        caching_str = str(self.caching_params)
+        caching_str = str(self.index_caching_params)
```
[the remark this answers is not in the document]
Comment (rahul-aerospike) [reply src/aerospike_vector_search/types.py]: updated.

File: tests/standard/sync/test_admin_client_index_get.py (hunk 1 above)
Comment (rahul-aerospike) [reply tests/standard/sync/test_admin_client_index_get.py:37]: added

File: tests/standard/aio/test_admin_client_index_list.py (hunk 2 above)
Comment (rahul-aerospike) [reply tests/standard/aio/test_admin_client_index_list.py:35]: added

File: tests/standard/aio/test_admin_client_index_get.py (hunk 3 above)
Comment (rahul-aerospike) [reply tests/standard/aio/test_admin_client_index_get.py:31]: added

**Review by dwelch-spike — APPROVED** [review]
Nice! LGTM

**Pull request merged**